src/connectors/infinite-hits/connectInfiniteHits.js

```diff
@@ -96,29 +116,30 @@ export default function connectInfiniteHits(renderFn, unmountFn) {
       },
 
       render({ results, state, instantSearchInstance }) {
-        if (state.page === 0) {
```

Comment:
We removed this part of the implementation but do we find why it was implemented?

Reply:
Not sure why (seems to be here from the first Instant Hits implementation https://github.com/algolia/instantsearch.js/commit/cdf867544a9acb9a4973f700fdc5230393ca3d51).

It was flushing the cache if you were reaching the first page. But even in the original implementation the widget had a weird behavior with this if you were using your browser to go back to previous pages: when reaching the first page the results list was emptied and only the last page was displayed.

This was even more problematic with the previous button, so I removed it.

Reply:
it's indeed already in the initial implementation: https://github.com/algolia/instantsearch.js/pull/1961/files#diff-db508ea4bf1e589da868d7cc605ac30bR77